cost/aws/superseded_instance/README.md

```diff
@@ -0,0 +1,34 @@
+## AWS Superseded Instance Types Remediation Policy
+
+### What it does
+
+This Policy Template is used to automatically resize Amazon Resource Manager Instances (VMs) based on user-defined standards.  For example, the Policy allows for a Disallowed Instance Type and its corresponding Superseded Instance Type.  If any Instances are found using the Disallowed type, they will be included in the corresponing Policy Incident.  Upon approval, Cloud Workflow will resize the Instance to the Superseded type. 
```

Comment:
What is Amazon Resource Manager.  I think it should just say AWS .

Reply:
this sentence isn't clear. `For example, the Policy allows for a Disallowed Instance Type and its corresponding Superseded Instance Type.`   allows for what?